javascript-light-bulb/main.js

```diff
@@ -0,0 +1,15 @@
+let clicks = 0;
+const lit = document.querySelector('.light');
+const back = document.querySelector('.div');
+
+function handleClick(event) {
+  clicks++;
+  if (clicks % 2 === 1) {
+    lit.className = 'light off';
+    back.className = 'div o';
+  } else {
+    lit.className = 'light on';
+    back.className = 'div l';
+  }
+}
```

Comment:
Interesting solution! Another way of solving for this is to have a boolean variable (named something like `isOn` set to false on line 1. Then the `handleClick` function could toggle the value from `false` to `true` and vise versa each time the button is clicked, and the classes can be set based on the value of `isOn`.

I'll go over the solution in class.